stack.yaml

```diff
@@ -1,3 +1,4 @@
+# Something is wrong with the new cardano-prelude resolver @ e0c6e29
 resolver: https://raw.githubusercontent.com/input-output-hk/cardano-prelude/00487726c4bc21b4744e59d913334ebfeac7d68e/snapshot.yaml
```

Comment:
For some reason this doesn't `stack build` when using `cardano-prelude@e0c6e29`. Looking into it.